src/workloads/workloads.csproj

```diff
@@ -0,0 +1,185 @@
+﻿<Project DefaultTargets="Restore;Build">
```

Comment:
Restore and Build targets should never run in the same node and share their evaluation as restore impacts properties and items in the Build target's evaluation.

I don't see why this line at all is necessary, I would just remove the DefaultTargets.

Reply:
When an msbuild project doesn't compile any C# code it shouldn't be named `.csproj`. Instead we use the `.proj` suffix for such.

Also I recommend to use the `Microsoft.Build.NoTargets` Sdk here instead which allows to override the Build target in a less hacky way.